test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -226,6 +227,32 @@ call_log_manager::call_log_set_timestamp(const json &call_log_entry)
           "', and config: '" + config + "'";
 }
 
+void
+call_log_manager::call_log_timestamp_transaction_uint(const json &call_log_entry)
+{
+    /* Convert the transaction type char * to a string object. */
+    const std::string wt_ts_txn_type = call_log_entry["input"]["wt_ts_txp_type"].get<std::string>();
+
+    /*
+     * There are no timestamps to be set if the timestamp type is not specified.
+     */
+    if (wt_ts_txn_type == "unkown")
```

Comment:
I think it is better we have an error msg here, to indicate that timestamp hasn't been set.

Reply:
From my understanding, wiredtiger just silently ignores this case, so there wouldn't be any particular message to compare the output to. Should I still add a message regardless?